blobfile/_context.py

```diff
@@ -217,11 +232,18 @@ def glob(self, pattern: str, parallel: bool = False) -> Iterator[str]:
                 raise Error("Advanced glob queries are not supported")
             yield from _local_glob(pattern)
         else:
-            for entry in self.scanglob(pattern=pattern, parallel=parallel):
+            for entry in self.scanglob(
+                pattern=pattern,
+                target_parallelism=target_parallelism,
+                parallel_char_set=parallel_char_set,
+            ):
                 yield entry.path
 
     def scanglob(
```

Comment:
I'm considering leaving the original parallel implementation around, as there are cases where the old scanglob parallel was much faster than this new one. This occurs when there's long shared prefixes. Ex. if you have a query like `/a/**/*.gz`, and the files are actually under `/a/really_long_shared_prefix/files.gz`, the parallelizer will take awhile to figure out `really_long_shared_prefix`.